docs/developer/api-conventions/prices.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Prices
+sidebar_label: Prices
+---
+
+## Introduction 
+
+Saleor API have 4 different types for prices, depending on taxes and eventual price range. 
+Althrough using [ISO 4217](https://pl.wikipedia.org/wiki/ISO_4217) codes for currency is recommended, we allow using custom ones. 
+
+## Frontend integration
+
+Displaying prices vary in different regions. If you are building your own storefront, you can use [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) if you plan to translate your application to different languages.
+
+
+# Api Reference
+
+### Money
+[`Money`](developer/api-reference/objects/money.mdx) is a basic type for keeping prices without taxes. We do not restrict currency code values, so you can even use cryptocurrencies or other custom currency.
+
+```graphql
+type Money {
+  currency: String!
+  amount: Float!
+}
+```
+
+- `currency`: Currency code for price. Currency depends on chosen Channel.
+- `amount`: Price amount.
+
+## TaxedMoney
+Most used example of [`TaxedMoney`](developer/api-reference/objects/taxed-money.mdx) is [`ProductVariant`](developer/api-reference/objects/product-variant.mdx) price. Even if you do not use any taxes integration, both net and gross prices will be populated with the same value.
+
+```graphql
+type TaxedMoney {
+  currency: String!
+  gross: Money!
+  net: Money!
+  tax: Money!
+}
+```
+
+- `currency`: Currency code for price. Currency depends on chosen Channel.
+- `gross`: Gross value (price with taxes).
+- `net`: Net value (price without taxes).
+- `tax`: Tax value.
+
+## MoneyRange
+[`MoneyRange`](developer/api-reference/objects/money-range.mdx) is used in displaying price for [`ShippingZone`](developer/api-reference/objects/shipping-zone.mdx).
+
+```graphql
+type MoneyRange {
+  start: Money
+  stop: Money
+}
+```
+
+- `start`: Lowest price in the range.
+- `stop`: Highest price in the range.
+
+## TaxedMoneyRange
+[`TaxedMoneyRange`](developer/api-reference/objects/taxed-money-range.mdx) is used in [`Product`](developer/api-reference/objects/product.mdx), because variants can differ in prices. You can display it as "price starts from $10" or "$10-$20".
```

Comment:
```suggestion
[`TaxedMoneyRange`](developer/api-reference/objects/taxed-money-range.mdx) is used in [`Products`](developer/api-reference/objects/product.mdx), because variants can differ in prices. You can display it as "price starts from $10" or "$10-$20".
```